beacon-chain/operations/voluntaryexits/service.go

```diff
@@ -99,7 +99,7 @@ func (p *Pool) InsertVoluntaryExit(ctx context.Context, state *beaconstate.Beaco
 func (p *Pool) MarkIncluded(exit *ethpb.SignedVoluntaryExit) {
 	p.lock.Lock()
 	defer p.lock.Unlock()
-	exists, index := existsInList(p.pending, exit.Exit.ValidatorIndex)
+	exists, index := existsInList(p.pending, uint64(exit.Exit.ValidatorIndex))
```

Comment:
Wouldn't it be better if we updated `existInList` to (`searchingFor` is index, so let's type it as such):
```golang
func existsInList(pending []*ethpb.SignedVoluntaryExit, searchingFor types.ValidatorIndex) (bool, int) 
```
Then conversions inside `existsInList` will be unnecessary (that's `return uint64(pending[j].Exit.ValidatorIndex) >= searchingFor` will be just `return pending[j].Exit.ValidatorIndex >= searchingFor`).